docker-compose/is/docker-compose.yml

```diff
@@ -1,34 +1,53 @@
-version: '2.3'
+version: '3.7'
 services:
-  mysql:
-    image: mysql:5.7.29
-    ports:
-      - 3306
-    environment:
-        MYSQL_ROOT_PASSWORD: root
-        MYSQL_USER: wso2carbon
-        MYSQL_PASSWORD: wso2carbon
-    volumes:
-      - ./conf/mysql/scripts:/docker-entrypoint-initdb.d
-      - ./conf/mysql/conf/my.cnf:/etc/mysql/my.cnf
-    healthcheck:
-      test: ["CMD", "mysqladmin" ,"ping", "-uroot", "-proot"]
-      interval: 30s
-      timeout: 60s
-      retries: 5
-  identity-server:
-    build: ./dockerfiles/is
-    ports:
-      - "9763:9763"
-      - "9443:9443"
-    healthcheck:
-      test: ["CMD", "curl", "-k", "-f", "https://localhost:9443/carbon/admin/login.jsp"]
-      interval: 5s
-      timeout: 120s
-      start_period: 100s
-    volumes:
-      - ./conf/identity-server:/home/wso2carbon/wso2-config-volume
-    depends_on:
-      mysql:
-        condition: service_healthy
+  # mysql:
+  #   image: mysql:5.7.29
+  #   ports:
+  #     - "3306"
+  #   environment:
+  #       MYSQL_ROOT_PASSWORD: root
+  #       MYSQL_USER: wso2carbon
+  #       MYSQL_PASSWORD: wso2carbon
+  #   volumes:
+  #     - ./conf/mysql/scripts:/docker-entrypoint-initdb.d
+  #     - ./conf/mysql/conf/my.cnf:/etc/mysql/my.cnf
+  #   healthcheck:
+  #     test: ["CMD", "mysqladmin" ,"ping", "-uroot", "-proot"]
+  #     interval: 30s
+  #     timeout: 60s
+  #     retries: 5
 
+  # mailserver:
```

Comment:
The mail server settings were added to deployment.toml but the mailserver is commented out here.